lib/krane/cli/version_command.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+require 'active_support/concern'
+
+module Krane
+  module VersionCommand
```

Comment:
Given the folder structure I'm wondering if this should be `module krane; module CLI; module VersionCommand` and then have the `Krane::CLI::Krane` instead of `Krane::CLI`

Reply:
I think that makes sense, since `Krane` (top-level) is the entire repo, including API accessible elements, whereas `Krane::CLI::Krane` is the actual scoping of the command-line entrypoint. It's a bit wordy but makes the separation of responsibilities clearer IMO

Reply:
Fixed